README.md

```diff
@@ -190,21 +190,24 @@ import "github.com/bytedance/sonic/decoder"
 var data interface{}
 err := sonic.UnmarshalString("[[[}]]", &data)
 if err != nil {
-    /*one line by default*/
+    /* One line by default */
     println(e.Error()) // "Syntax error at index 3: invalid char\n\n\t[[[}]]\n\t...^..\n"
-    /*pretty print*/
+    /* Pretty print */
     if e, ok := err.(decoder.SyntaxError); ok {
         /*Syntax error at index 3: invalid char
 
             [[[}]]
             ...^..
         */
         print(e.Description())
-    }else if me, ok := err.(decoder.MismatchTypeError); ok {
-        print(e.Description()
+    } else if me, ok := err.(decoder.MismatchTypeError); ok {
```

Comment:
the error should be a pointer `*decoder.MismatchTypeError`

Reply:
resolved in latest squashed commitset